resources/styles/components/exercise/step-card.less

```diff
@@ -33,6 +33,20 @@
     font-size: 1.2rem;
     line-height: 1.2rem;
   }
+
```

Comment:
can the tabbing in the below added styles be consistent with the rest of the file?